deploy/rinkeby/0092_liquidity_rewards.ts

```diff
@@ -0,0 +1,135 @@
+import { ethers as ethershardhat, hardhatArguments } from "hardhat";
+import { deployments } from "hardhat";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+require("dotenv").config();
+
+module.exports = async (hre: HardhatRuntimeEnvironment) => {
+	let initial_run = process.env.INITIAL_RUN == "false" ? true : false;
+
+	if (hardhatArguments.network === "rinkeby" && initial_run) {
+		const ethers = hre.ethers;
+		const namedAccounts = await hre.getNamedAccounts();
+		const deployer = namedAccounts.deployer;
+		const { log } = deployments;
+		let OrchestratorDeployment = await deployments.get("Orchestrator");
+		let ctxDeployment = await deployments.get("Ctx");
+		let timelock = await deployments.get("Timelock");
+
+		let vestingRatio = process.env.VESTING_RATIO;
```

Comment:
need to add to `env.sample`?

Reply:
[Not for this PR] But I just want to make sure that we'll be using the reward smart contract on the mainnet that is already deployed.  We simply need to call `notifyRewardAmount`